responsibleai/responsibleai/serialization_utilities.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import datetime
+import numpy as np
+
+from typing import Any
+
+
+def serialize_json_safe(o: Any):
```

Comment:
this looks like:
C:\responsible-ai-widgets\responsibleai\responsibleai\_input_processing.py
except it is missing forObject parameter which Ke used.  Why did we decide to move it to a separate file and modify it like this?

Reply:
There is a copy of this function in interpret-community as well. https://github.com/interpretml/interpret-community/blob/c6fe5c2436f2316bcf976acd45d827aeb66d99f4/python/interpret_community/common/serialization_utils.py#L11. We can import the function from interpret-community instead. Why are we having so copies of the same fuinction..

Reply:
I think the reason originally was that no one depended on anyone else -- raiwidgets didn't depend on interpret-community and interpret-community didn't depend on raiwidgets

Reply:
It needs to be made public for use in other packages. As discussed offline I'll update to handle the `__dict__` differently.

Reply:
Fixed